structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,125 @@
+4. Once the Collaborators informally agree that enough information has been gathered, the vote can be created with the option to approve or deny, according to the process defined in this document. The Invitees should be informed the vote has begun.
+5. After the poll is closed, the vote is considered passed if more than 75% of the votes cast are positive. The discussion channel must be archived and kept read-only.
+- If the vote has passed: the Invitees are elevated to the rank of Collaborators. Internal discussions can be purged first if needs be. The community is added to any listing that may exist.
+- If the vote is rejected: enough time is given to the Invitees to acknowledge the result and leave the server. Managers are free to kick Invitees if necessary.
+
+# Rationale and alternatives
+
+The Quilt Community Collab program existed prior to this document. This document tries to summarize how the server has been working so far while formalising some aspects that were still left up in the air.
+
+More importantly, three polls have been held inside the server, with the following results:
+
+- When voting for changes on who is or isn't part of the CC network, how should we count votes?
+    - 10 answered: One vote per community: each community will have to write down a formal process on which they decide to vote for or against changes. This process must be democratic and fair.
+    - 4 answered: One vote per collaborator: each user or plural system present in the server gets one vote
+- When voting on who is or isn't part of the CC network, what proportions of approvals versus neutral votes are required to pass? (multiple answers accepted)
+    - 11 answered: >75% approval to pass (there must be at least 3 times more approval than neutral)
+    - 4 answered: >50% approval to pass (there must be more approval than neutral)
+    - 1 answered: 100% approval to pass (there are no neutral votes, only vetos)
+- When voting for changes on who is or isn't part of the CC network, should we have veto rights, and if so how should they work?
+    - 17 answered: Any voter can veto, which can then be overruled by the majority if the concerns aren't believed to be deserved
+    - Nobody answered: There are no veto rights.
+    - Nobody answered: Any voter can veto, which will immediately cancel the vote
+
+# Prior art
+
+Quilt Community Collab is an indirect replacement of the Modding Interpol, with a wider range of activities, such as events and related projects.
+
+# Unresolved questions
+
+The following questions will have to be solved at a later date:
+
+- This currently doesn’t define how to remove communities
+- We aren’t defining any role for administrators of the communities part of the program
+- We only allow for full communities to join, we should look at whenever we also want to invite individuals
```

Comment:
What examples would we have of individuals being invited to the program in their personal capacity (rather than in a representative capacity for a community)?

Reply:
At the moment we do not, but we might want to add some individual members, which is why this is a future concern